terminus-ui/login-form/src/login-form.component.html

```diff
@@ -15,6 +15,7 @@
     [isRequired]="true"
     [spellcheck]="false"
     [autocapitalize]="false"
+    autocomplete="email"
```

Comment:
what difference does it make with or without this `autocomplete` input?

Reply:
The `autocomplete` attribute tells the browser what data should be used when providing autocompletion.

In my quick tests it seems that both Chrome and FF can infer this if the input has a name or type that defines `email`. But since this is the 'correct' way to define it, I thought it should be added.